code/ai/aicode.cpp

```diff
@@ -572,8 +572,17 @@ int ai_maybe_autoscale(int absolute_index)
  */
 void init_ai_class(ai_class *aicp)
 {
-	int i;
-	for (i = 0; i < NUM_SKILL_LEVELS; i++)
+
+	// AI Class main attributes
+	// Set the lowest values here so the table can be modular
+	for (int i = 0; i < NUM_SKILL_LEVELS; i++) {
+		aicp->ai_accuracy[i] = FLT_MIN;
+		aicp->ai_evasion[i] = FLT_MIN;
+		aicp->ai_courage[i] = FLT_MIN;
+		aicp->ai_patience[i] = FLT_MIN;
+	}
+
+	for (int i = 0; i < NUM_SKILL_LEVELS; i++)
 	{
```

Comment:
Why are these two loops, why not merge them into one?

Reply:
Eh, I was probably trying to change as little code as possible. Merged.